### ISSUE TITLE
Infenrence FPS is not normal

### ISSUE THREAD

**Issue by Jessespace** [dmlc/gluon-cv#542]
I have test the FPS in inference stage.
shape: 608*811   FPS: 62.35
shape: 416*555   FPS: 65.17
shape: 320*427   FPS: 65.04
When I input the 320*427 image,the FPS should be less than 416*555.And they have not much difference,is there any wrong?
```
num=len(read_paths)
toc=0.0
sum=0.0
for path in read_paths:
        img_path=os.path.join(read_dir,path)
        x, img = data.transforms.presets.yolo.load_test(img_path,short=shape)
        x = x.as_in_context(ctx)
        print('Shape of pre-processed image:', x.shape)
        tic = time.time()
        class_IDs, scores, bounding_boxs = net(x)
        toc =time.time()-tic
        sum = sum+ toc
print("every picture time cost:{:.2f}".format(float(num)/(sum)))
```

**Comment by zhreshold** [comment]
Check the data loding FPS, see if it's the bottleneck

**Comment by Jessespace** [comment]
> Check the data loding FPS, see if it's the bottleneck

Sorry,I don't understand your answer

**Comment by Jerryzcn** [comment]
Basically, how fast you can load the data
